admin/views/class-yoast-feature-toggle.php

```diff
@@ -0,0 +1,111 @@
+<?php
+/**
+ * WPSEO plugin file.
+ *
+ * @package WPSEO\Admin
+ */
+
+/**
+ * Class representing a feature toggle.
+ */
+class Yoast_Feature_Toggle {
+
+	/** @var string Feature toggle identifier. */
+	protected $name = '';
```

Comment:
`name` isn't in line with the `identifier` in the doc above.

Reply:
This is due to backward-compatibility. The class is new, but it essentially wraps data that has already existed and was filterable, so it's safe to stick with `$name`.